packages/medusa/src/api/routes/store/products/list-products.ts

```diff
@@ -199,6 +199,12 @@ export default async (req, res) => {
 
   // get only published products for store endpoint
   filterableFields["status"] = ["published"]
+  // store APIs only receive active and public categories to query from
+  filterableFields["categories"] = {
+    ...(filterableFields.categories || {}),
+    is_internal: false,
```

Comment:
comment: This seems to be consistent with my other comment.

Reply:
Didn't quite understand. Can you elaborate on this one @olivermrbl? 

I'm setting the scope of public `(is_internal: false)` AND active `(is_active: true)`

Reply:
It was a comment that supported my question on scopes. But now that you've clarified that, this is not relevant :)